components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/util/TenantMgtUtil.java

```diff
@@ -774,4 +779,22 @@ private static boolean isFormatCorrect(String regularExpression, String domainNa
         Matcher m2 = p2.matcher(domainName);
         return m2.matches();
     }
+
+    /**
+     * Triggers post tenant delete for TenantMgtListener
+     *
+     * @param tenantUuid String tenant unique identifier
+     * @param userUuid String user unique identifier
```

Comment:
Lets mention this is admin user uuid

Reply:
fixed 45acae978b872ecc6afaa3061417abb6164da3e0